### DISCUSSION TITLE
Tianji update failed

### DISCUSSION THREAD

**Discussion by Narkowii** [tteck/Proxmox#3782] [Help Needed] [answered]
Hi,

I've got an failed update with the Tianji update script.

`✓ Stopped Tianji Service
 /fatal: not a git repository (or any of the parent directories): .git

[ERROR] in line 71: exit code 0: while executing command git checkout -q v${RELEASE}`

**Comment by Narkowii** [comment]
Script updated
It works now
thanks

**Comment by futurepr0n** [comment]
Any chance you know how long it took to install from scratch? I ran the script but Extreme Patience, its been several hours now im wondering if anything is wrong or this is nornal

**Comment by Narkowii** [comment]
impossible to update Tianji v1.15.8 to v1.16

 **✓ Stopped Tianji Service
 /fatal: not a git repository (or any of the parent directories): .git

[ERROR] in line 71: exit code 0: while executing command git checkout -f -q `v${RELEASE}``**

**Comment by Narkowii** [comment]
always impossible to update Tianji v1.15.8 to v1.16.1

✓ Stopped Tianji Service
 /fatal: not a git repository (or any of the parent directories): .git

[ERROR] in line 69: exit code 0: while executing command git checkout -f -q v${RELEASE}

**Reply by tteck** [reply]
~Did it ask about resources? If not, it's still using cache.~

Sorry, wrong discussion.

**Comment by Narkowii** [comment]
Hello
Someone got the same error while trying to update ?

` ✓ Stopped Tianji Service
 /fatal: not a git repository (or any of the parent directories): .git

[ERROR] in line 69: exit code 0: while executing command git checkout -f -q v${RELEASE}`

**Reply by tteck** [reply]
@MickLesk can you take a look at this

**Reply by MickLesk** [reply]
ive pulled an tweak. if you push it, @Narkowii can test it. 
Ive setup the 1.16.3 and update with this script to 1.16.4

the creator of tianji has removed the .git files in the zip, thus no more pull possible

**Reply by Narkowii** [reply]
i am actually on 1.15.8 and update script give me this 

`   Setting Container to Higher Resources   environment: line 65: CTID: unbound variable
`

**Reply by MickLesk** [reply]
Wait for next pull

**Reply by MickLesk** [reply, marked as answer]
now, test again

**Reply by Narkowii** [reply]
got this error 

`✓ Stopped Tianji Service
 /rm: missing operand 1.16.4   
Try 'rm --help' for more information.

[ERROR] in line 74: exit code 0: while executing command rm -R`

**Reply by Narkowii** [reply]
`✓ Stopped Tianji Service
 ✓ Updated Tianji to 1.16.4
 ✓ Started Tianji
 ✓ Cleaned
 ✓ Updated Successfully`


THANK YOU
